### PR TITLE
Add examples for gradient repeat modes for linear and radial

### PR DESCRIPTION
**Pull request by drott**
Moving more examples from the SVG based test font to the scripted one,
so I can use a single combined test font in the Skia GM test.

### CODE REVIEW

File: config/more_samples-glyf_colr_1.py [hunk 1]
```diff
@@ -32,6 +34,11 @@ class SampleGlyph(NamedTuple):
     colr: Optional[Mapping[str, Any]] = None
 
 
+def _accessors():
+    for c in chain(range(ord("A"), ord("Z") + 1), range(ord("a"), ord("z") + 1)):
```
Comment (anthrotype) [inline config/more_samples-glyf_colr_1.py:38]: you could `yield from string.ascii_letters` constants defined in the `string` built-in module: https://docs.python.org/3/library/string.html#string.ascii_letters

File: config/more_samples-glyf_colr_1.py [hunk 2]
```diff
@@ -269,7 +288,75 @@ def _paint_scale(scale_x, scale_y, center_x, center_y, accessor_char):
...
+
+
+def _extend_modes(gradient_format, extend_mode, accessor_char):
+
+    format_map = {
+        "linear": ot.PaintFormat.PaintLinearGradient,
+        "radial": ot.PaintFormat.PaintRadialGradient,
+    }
+
+    if gradient_format not in format_map.keys():
```
Comment (anthrotype) [inline config/more_samples-glyf_colr_1.py:303]: don't need `.keys()` to check containment in a dict
```suggestion
    if gradient_format not in format_map:
```

**Review by anthrotype — APPROVED** [review]
LGTM

File: config/more_samples-glyf_colr_1.py (hunk 2 above)
Comment (drott) [reply config/more_samples-glyf_colr_1.py]: Done in two cases, thanks for the suggestion.

File: config/more_samples-glyf_colr_1.py (hunk 1 above)
Comment (drott) [reply config/more_samples-glyf_colr_1.py]: Done, and this actually revealed above in how I used yield and the generator, should be fixed now.

File: config/more_samples-glyf_colr_1.py [hunk 3]
```diff
@@ -378,28 +385,29 @@ def main():
         "psName": "-".join((_FAMILY.replace(" ", ""), _STYLE)),
     }
 
+    access_chars = _accessors()
```
Comment (anthrotype) [inline config/more_samples-glyf_colr_1.py:388]: you could even do without the _accessors() function, and simply do `access_chars = iter(string.ascii_letters)` here

**Review by anthrotype — APPROVED** [review]

File: config/more_samples-glyf_colr_1.py (hunk 3 above)
Comment (drott) [reply config/more_samples-glyf_colr_1.py]: Done, thanks

**Comment by drott** [comment]
Thanks for the quick review.

**Pull request merged**